spec.bs

```diff
@@ -589,7 +630,8 @@ A <dfn>Redemption Record</dfn> is a [=byte sequence=].
 
 To <dfn>get an unexpired [=Redemption Record=]</dfn> with an [=/origin=] |issuer| and an [=/origin=] |topLevel|:
 
-1. TODO
+1. Let |record| be the result of performing [=retrieve a redemption record=] with |issuer| and |topLevel|.
```

Comment:
I'm fine either way, but you could also just use the retrieve algorithm directly in https://wicg.github.io/trust-token-api/#ref-for-get-an-unexpired-redemption-record